Kazi27/moodboard-frontend
# Implemented PUT/DELETE requests from backend

Implemented PUT/DELETE requests from backend

Testing Instructions: 
1. Run npm start on both the front-end repo and back-end repo
2. Navigate to edit favorites page and ensure favorited photos appear
3. Click on the edit icon of a favorited image and edit caption and/or image url to test PUT request
4. Click on the delete icon of a favorited image to test DELETE request
